dcos/errors.py

```diff
@@ -49,11 +49,17 @@ class DCOSAuthenticationException(DCOSHTTPException):
 
     :param response: requests Response object
     :type response: Response
+    :param message: An optional message for the exception.
```

Comment:
I wouldn't assume it at the class definition level, as it depends on how it's used. Maybe I can turn the sentence like this?

> An optional message for the exception. This can be useful to provide the user with more meaningful information.